Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM fedora:34
 
+RUN curl https://packages.codenotary.org/codenotary.repo -o /etc/yum.repos.d/codenotary.repo
 RUN mkdir -p /code && \
     yum update -y && \
-    yum install python3-virtualenv python39 libmodulemd python3-libmodulemd \
-                python3-libmodulemd1 modulemd-tools python-gobject -y && \
+    yum install --enablerepo="codenotary-repo" cas python3-virtualenv python39 libmodulemd python3-libmodulemd \
+                python3-libmodulemd1 modulemd-tools python-gobject git -y && \
```

Comment:
Question. Do we need git?

Reply:
yeah, we need to install `git` for installing `cas_wrapper`